src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,35 @@
+package view;
+
+import board.GameBoard;
+import direction.Point;
+import java.util.Optional;
+import piece.Piece;
+import piece.Pieces;
+
+public class OutputView {
+    public static void displayBoard(GameBoard gameBoard) {
+        Pieces pieces = gameBoard.findAllPieces();
+
+        System.out.println();
+        for (int i = 1; i <= 10; i++) {
+            for (int j = 1; j <= 9; j++) {
+                Point point = new Point(j, i);
+                Optional<Piece> findPiece = pieces.findByPoint(point);
+                if(findPiece.isEmpty()) {
+                    System.out.print(".");
+                    continue;
+                }
+                System.out.print(findPiece.get().getNickName());
+            }
+            System.out.println(" " + i);
+        }
+        System.out.println("123456789");
+        System.out.println();
+    }
+
+
+
+    public static void displayWrongPoint() {
+        System.out.println("본인의 기물이 아닙니다. 다시 선택해 주세요.");
+    }
+}
```

Comment:
[코멘트](https://github.com/woowacourse/java-janggi/pull/66#discussion_r2009736604)에 대한 답변 이곳에 남길게요.

불필요 공백 제거 및 if ()가 붙어있어서 말씀 드렸어요!
```suggestion
    public static void displayBoard(GameBoard gameBoard) {
        Pieces pieces = gameBoard.findAllPieces();

        System.out.println();
        for (int i = 1; i <= 10; i++) {
            for (int j = 1; j <= 9; j++) {
                Point point = new Point(j, i);
                Optional<Piece> findPiece = pieces.findByPoint(point);
                if (findPiece.isEmpty()) {
                    System.out.print(".");
                    continue;
                }
                System.out.print(findPiece.get().getNickName());
            }
            System.out.println(" " + i);
        }
        System.out.println("123456789");
        System.out.println();
    }

    public static void displayWrongPoint() {
        System.out.println("본인의 기물이 아닙니다. 다시 선택해 주세요.");
    }
}
```

Reply:
헉 수정 완료하였습니다..! 코드 포맷팅을 계속해서 신경쓰지 못하고 있네요. 조금 더 의식적으로 노력하겠습니다!